package.json

```diff
@@ -63,6 +63,7 @@
     "typescript-plugin-css-modules": "^5.0.2"
   },
   "peerDependencies": {
+    "@eveshipfit/data": "*",
```

Comment:
Maybe we are better off pinning this too, to indicate what version we tested on?
At least it makes it in line with dogma engine?